jasmine/tests/failing_shard.test.js

```diff
@@ -0,0 +1,8 @@
+describe('test failures with sharding', () => {
```

Comment:
Should only 1/2 of the shards fail?

Reply:
nope. they both fail.

Reply:
I mean making only 1 shard fail and ensuring bazel still considers that a failure, not only when every shard fails...

Reply:
Ah. i just took this test from rules_nodejs.  We could add that case later.